Z64 Utils/Z64/Z64Object.cs

```diff
@@ -968,6 +1028,7 @@ public string GetJSON()
                             });
                             break;
                         }
+                    case EntryType.Mtx:

```

Comment:
`EntryType.Mtx` should also be handled in the `FromJson` function.
Adding this should work:
```
case EntryType.Mtx:
{
    obj.AddMtx(1);
    break;
}
```